pkg/gadgets/tcptop/tracer/tcptop_bpfel.go

```diff
@@ -62,7 +62,6 @@ type tcptopProgramSpecs struct {
 //
 // It can be passed ebpf.CollectionSpec.Assign.
 type tcptopMapSpecs struct {
-	CgroupMap  *ebpf.MapSpec `ebpf:"cgroup_map"`
```

Comment:
Just out of curiosity: What happened here? An old version of this file was committed? :thinking:

Reply:
I think it is because it wasn't updated after cgroup map was removed.